geode-redis/src/integrationTest/java/org/apache/geode/redis/GeoIntegrationTest.java

```diff
@@ -69,9 +68,18 @@ public static void setUp() throws IOException {
 
   @After
   public void cleanup() {
+    // TODO: Correct implementation of flushAll so it deletes Geo* keys properly and this zrem is
+    // not needed
```

Comment:
Can we please create a Jira ticket to track the todos?

Reply:
Created  GEODE-7909, updated TODO comments to refer to Jira.